graphkb/constants.py

```diff
@@ -172,3 +172,45 @@ def __getitem__(self, key):
     "mutations": "mut",
     "nonsense": ">",
 }
+
+# For match.type_screening() [KBDEV-1056]
+DEFAULT_NON_STRUCTURAL_VARIANT_TYPE = 'mutation'
+STRUCTURAL_VARIANT_SIZE_THRESHOLD = 50  # bp
+STRUCTURAL_VARIANT_TYPES = [
+    "structural variant",
+    "insertion",
+    "in-frame insertion",
+    "deletion",
+    "deletion polymorphism",
+    "in-frame deletion",
+    "translocation",
+    "inverted translocation",
+    "inversion",
+    "indel",
```

Comment:
I think 'indel' is usually used to refer to 'small' insertions and deletions.

Reply:
In the GraphKB Vocabulary tree we use for variant types, 'indel' is linked to both 'structural variant' and 'small mutation' terms:
![image](https://github.com/bcgsc/pori_graphkb_python/assets/4022327/20974404-8387-474e-8a61-3fa3628c8a14)
If you think they should never be considered as structural no matter the size, I'll add a line of code for that, np, just let me know. We should then also see if the Vocabulary tree should be updated accordingly.